daemons/lvmdbusd/cmdhandler.py

```diff
@@ -164,7 +164,7 @@ def my_callback(my_context, line_read_stdin)
 			break
 
 	if process.returncode is not None:
-		if debug or process.returncode != 0:
+		if debug or (process.returncode != 0 and (process.returncode != 5 and "fullreport" in command)):
```

Comment:
Are you sure this happens only for exported VGs? LVM returns 5 for most errors so this might hide another issue.

Reply:
I'll double check with lvm folks, thanks

Reply:
In all cases the command in is required to be `fullreport`.  This code path is used internally by the daemon, thus we shouldn't be hiding anything.  This specific change is to reduce debug logging.  There are other changes that were put in place in 2018 for the exitcode `5` on fullreport.